x/epochs/types/telemetry.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import "github.com/osmosis-labs/osmosis/osmoutils/observability"
+
+var (
+	// epoch_hook_failed
+	//
+	// counter that is increased if epoch hook fails
+	//
+	// Has the following labels:
+	// * module_name - the name of the module that errored or panicked
+	// * err - the error or panic returned
+	// * is_before_hook - true if this is a before epoch hook. False otherwise.
+	EpochHookFailedMetricName = formatEpochMetricName("hook_failed")
```

Comment:
@alessandrolomanto could you please review all `telemetry.go` update files in this PR and configure alerts for each?

If the counter increases, it should send an alert. Please also format labels in the alert